elliottlib/cli/tarball_sources_cli.py

```diff
@@ -136,7 +136,7 @@ def create(ctx, advisories, out_dir, out_layout, components, force):
 
                 LOGGER.debug("Copying {} to {}...".format(
                     temp_tarball_path, tarball_abspath))
-                shutil.copy2(temp_tarball_path, tarball_abspath)
+                shutil.copyfile(temp_tarball_path, tarball_abspath)  # `shutil.copyfile` uses default umask
```

Comment:
@vfreex how do I know this shutils.copyfile make the file (604) I suppose is the permission RCM wants

Reply:
Updated the rsync instruction